ecs-agent/daemonimages/csidriver/mounter/safe_mounter_unix.go

```diff
@@ -0,0 +1,29 @@
+//go:build linux || darwin
+// +build linux darwin
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package mounter
+
+import (
+	"k8s.io/mount-utils"
+	utilexec "k8s.io/utils/exec"
+)
+
+func NewSafeMounter() (*mount.SafeFormatAndMount, error) {
+	return &mount.SafeFormatAndMount{
+		Interface: mount.New(""),
```

Comment:
Why is `""` used for `mounterPath` parameter?

Reply:
Not sure. Following the existing EBS csi driver implementation.

Reply:
The implementation will use the default mount command.